src/blas/backends/rocblas/rocblas_task.hpp

```diff
@@ -55,7 +55,7 @@ static inline void host_task_internal(H &cgh, sycl::queue queue, F f) {
 #else
 template <typename H, typename F>
 static inline void host_task_internal(H &cgh, sycl::queue queue, F f) {
-    cgh.interop_task([f, queue](sycl::interop_handler ih) {
+    cgh.host_task([f, queue](sycl::interop_handle ih) {
```

Comment:
Could it be affected by changes in DPC++ compiler for dropping host support?

Reply:
@mkrainiuk In my tests with latest LLVM, I did not see any deprecation warnings related to host_task(). I guess, if/when the compiler drops support, we can switch to recommended API.